packages/app-page-builder/src/blockEditor/Editor.tsx

```diff
@@ -87,7 +84,10 @@ export const BlockEditor: React.FC = () => {
             <BlockEditorConfig />
             <LoadData>
                 <PbEditor
-                    stateInitializerFactory={createStateInitializer(block as BlockWithContent)}
+                    stateInitializerFactory={createStateInitializer(
+                        block as BlockWithContent,
+                        blockCategories as BlockCategoriesAtomType
```

Comment:
As noted in my previous comment, you won't need this, if you move the whole logic of loading block categories into the hook.